docs/kyma-environment-broker/08-06-orchestration-status.md

```diff
@@ -98,7 +95,7 @@ A successful call returns the list of all orchestrations:
    export ORCHESTRATION_ID={OBTAINED_ORCHESTRATION_ID}
    ```
 
-2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](#details-orchestration) to fetch the list of the upgrade operations for a given orchestration.
+2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](03-10-orchestration.md) to fetch the list of the upgrade operations for a given orchestration.
```

Comment:
I cannot see any request header details in the linked document. Did you mean `03-05-authorization.md`?